Dockerfile

```diff
@@ -1,31 +1,39 @@
-FROM python:3.8-slim-buster as builder
+FROM python:3.9-slim as base
 
 ENV LANG=C.UTF-8 \
     DEBIAN_FRONTEND=noninteractive \
     PIP_NO_CACHE_DIR=true
 
-RUN apt-get update && apt-get install --no-install-recommends -qq -y gcc g++ && apt-get clean autoclean \
-  && apt-get autoremove -y \
-  && rm -rf /var/lib/apt/lists/* \
-  && rm -f /var/cache/apt/archives/*.deb
```

Comment:
Is the new version a proper replacement for this one?

Reply:
Not sure what this is used for. After builded container check this folder and there is no deb files, so this command is redundant